docker/conda/environments/cuda11.8_examples.yml

```diff
@@ -34,3 +34,4 @@ dependencies:
     - mlflow>=2.2.1,<3
     - papermill=2.3.4
     - s3fs>=2023.6
+    - databricks-connect
```

Comment:
This dep doesn't need to be in both the dev and examples yaml files. 
Just the dev yaml should be fine

Reply:
sounds good, i'll make the changes